### PR TITLE
Check overlap time and confirmation when creating repeated schedule

### PR DESCRIPTION
**Pull request by vanlinha3k35pbc**

### CODE REVIEW

File: app/assets/javascripts/repeats.coffee [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      event.preventDefault()
+      $data = $("[class *=schedule]").serialize()
+      $.ajax
+        url: "/repeats/check_overlap_time"
+        type: "GET"
+        data: $data
+        success: (result) ->
+          overlap_time_days = result.overlap_time_days
+          if overlap_time_days.length > 0
+            retVal = confirm "Overlap Time Days: #{overlap_time_days} \nDo you want to continue ?"
```
Comment (vanlinha3k35pbc) [inline app/assets/javascripts/repeats.coffee]: Chưa dùng được I18n

Comment (dieunb) [inline app/assets/javascripts/repeats.coffee:17]: i18n???

File: app/assets/javascripts/repeats.coffee [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            if retVal == false
```
Comment (dieunb) [inline app/assets/javascripts/repeats.coffee:18]: why not `if !retVal`

File: app/assets/javascripts/repeats.coffee [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+              return false
+        $('[class *=schedule]').submit()
```
Comment (dieunb) [inline app/assets/javascripts/repeats.coffee:20]: use `'` instead of `"`

**Pull request closed without merging**